libbeat/outputs/elasticsearch/client.go

```diff
@@ -270,7 +284,13 @@ func bulkEncodePublishRequest(
 			log.Errorf("Failed to encode event meta data: %+v", err)
 			continue
 		}
-		bulkItems = append(bulkItems, meta, event)
+		opType := eventMetaValue(event, "op_type")
+		if opType == "delete" {
```

Comment:
Similar to making `"op_type"` a `const`, lets also make `const` for this `"delete"` bare string so we can give it a godoc comment. This will allow developers reading godocs for this package to discover that it's possible to delete events.